README.md

```diff
@@ -1,6 +1,8 @@
 # Mini
 [![Release](https://jitpack.io/v/hyperdevs-team/mini-kotlin.svg)](https://jitpack.io/#hyperdevs-team/mini-kotlin)
 
+## ⚠️ DOCS HAVE TO BE UPDATED FOR VERSION 2.x.y
```

Comment:
```suggestion
## ⚠️ DOCS HAVE TO BE UPDATED FOR VERSION 3.x.y
```

Reply:
Done in 839877ca927656bc40554e6dea7cbf83e22bdac1